graphkb/constants.py

```diff
@@ -172,3 +172,45 @@ def __getitem__(self, key):
     "mutations": "mut",
     "nonsense": ">",
 }
+
+# For match.type_screening() [KBDEV-1056]
+DEFAULT_NON_STRUCTURAL_VARIANT_TYPE = 'mutation'
+STRUCTURAL_VARIANT_SIZE_THRESHOLD = 50  # bp
```

Comment:
oh, really?
Erin commented on ticket:
> Our pipeline cutoff for SVs is often 50 bp, maybe we could apply that here? It is rather arbitrary in any case.

Do you know who could make the call on that one?

Reply:
Actually it usually is Erin.

I guess we can just go with this for now.
At the next clinical meeting, we'll try to bring up again:
 * The division between a 'small mutation' indel and 'Structural Variant' indel is completely arbitrary.
 * What is the cutoff, in base-pairs, between small and large?  (Keep protein size changes separate 1/3 size)
 * How does TC pipeline and everything else find this number to keep things consistent.
 * Since the difference a 'Small Mutation' and 'Structural Variant' is a completely arbitrary size cutoff, can we make these variants have the same properties?  eg. hgvs notations for Structural Variants.
 * Structural Variants have gene1/2, transcript1/2, exon1/2, n/ctermGene, n/ctermTranscript - pick 1/2 or cterm/nterm only and use consistently.